app/Commands/InspireCommand.php

```diff
@@ -41,11 +41,8 @@ public function handle()
 
     /**
      * Define the command's schedule.
-     *
-     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
-     * @return void
      */
-    public function schedule(Schedule $schedule)
+    public function schedule(Schedule $schedule): void
```

Comment:
The method handle should be typed too - to `void`.

Reply:
Hmm. I thought it could be `void` or `int`? (I have various commands where I return the status code)

Reply:
`void` for the inspire command.

Reply:
In other words, `void` by default. But people can change it to `int` if they want.

Don't forget to also type hint the documentation.

Reply:
🤦🏻 Thought this was in the framework, will make the changes tomorrow (later today). 👍🏻